src/hashtree.erl

```
@@ -631,7 +631,7 @@ get_disk_bucket(Level, Bucket, #state{id=Id, ref=Ref}) ->
 set_disk_bucket(Level, Bucket, Val, State=#state{id=Id, ref=Ref}) ->
     HKey = encode_bucket(Id, Level, Bucket),
     Bin = term_to_binary(Val),
-    eleveldb:put(Ref, HKey, Bin, []),
+    ok = eleveldb:put(Ref, HKey, Bin, []),
```

Indeed, my main reason for introducing this is to raise the questions: 
- Should the process continue if the put fails? Will this lead to incorrect results?
- If it is safe to ignore error conditions, why?